plugins/Tools/RotateTool/RotateTool.qml

```diff
@@ -45,6 +45,23 @@ Item
         onClicked: UM.ActiveTool.triggerAction("layFlat");
     }
 
+    Button
+    {
+        id: alignFaceButton
+
+        anchors.left: layFlatButton.right;
+        anchors.leftMargin: UM.Theme.getSize("default_margin").width;
+
+        text: catalog.i18nc("@action:button","Align selected face with bottom")
+        iconSource: UM.Theme.getIcon("rotate_layflat");  // TODO!
```

Comment:
There's a `TODO` here.

Reply:
Yes, because I don't have the actual logo yet. We should remember to make one before 4.3-final.

Reply:
Maybe @Yian-Lai can help with that.

Reply:
I'm not creating one now, because then there is a large chance that it'll stay there forever.

Reply:
@diegopradogesto Yes, but not _today_ ;-)